tests/copp/test_copp.py

```diff
@@ -191,12 +191,17 @@ def _gather_test_params(tbinfo, duthost, request):
                                topo=topo,
                                bgp_graph=bgp_graph)
 
-def _setup_testbed(dut, creds, ptf, test_params):
+@pytest.mark.usefixtures('disable_container_autorestart')
+def _setup_testbed(dut, creds, ptf, test_params, \
```

Comment:
If all autorestart of containers are disabled in pretest, will this fixture be able to enable it for lldp?

Reply:
Yes, it will enable the autorestart of the `feature_list` as part of `_teardown_testbed`.
But I think you are pointing out that we should not do this as the pretest disabled the autorestart permanently throughout the tests run.

In that case, I can add a check to verify if the autorestart was already disabled, and do not take any action to disable/enable this inside the test.
If the autorestart was enabled before the test, I let the test disable and enable it back. What do you think?

Reply:
I think it should be ok, it restores from a saved list.

Reply:
I see. Thanks.

Reply:
Disable the autostart fixture for an already disabled container should cause no change. https://github.com/Azure/sonic-buildimage/pull/5689 addressed that issue.